studio/components/salariesInput/components/SalariesTableEditor.tsx

```diff
@@ -0,0 +1,46 @@
+import styles from "../salariesInput.module.css";
+import SalaryNumberInput from "./SalaryNumberInput";
+import { Grid } from "@sanity/ui";
+import { Salaries } from "../utils/parseSalaries";
+
+interface SalariesTableEditorProps {
+  salaries: Salaries;
+  onYearSalaryChange: (year: string, salary: number) => void;
+}
+
+const SalariesTableEditor = ({
+  salaries,
+  onYearSalaryChange,
+}: SalariesTableEditorProps) => {
+  return (
+    <Grid columns={[2]}>
+      <div className={styles.tableHeader}>
+        <span className={styles.tableHeaderLabel}>Examination Year</span>
+      </div>
+      <div className={`${styles.tableHeader} ${styles.tableSalaryHeader}`}>
```

Comment:
For proper UU this should be a table with rows and columns I believe. but not entirely certain as there's an input in each row.. 🤔  Could you research this just a little for me?